FeedStoreChallenge/Infrastructure/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Vladimir Mironiuk on 04.09.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
```

Comment:
`ManagedCache` is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class ManagedCache: NSManagedObject {
```